src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -165,8 +191,8 @@ class boss_bronjahm : public CreatureScript
                             me->CastSpell(me, SPELL_SOULSTORM, false);
                             break;
                         case EVENT_FEAR:
-                            if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 1, 0.0f, true))
-                                me->CastCustomSpell(SPELL_FEAR, SPELLVALUE_MAX_TARGETS, 1, target, false);
+                            if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 0, 0.0f, true)) // Doesn't he fear just one enemy?
+                            DoCast(target, SPELL_FEAR);
```

Comment:
4 spaces added in front of `DoCast(target, SPELL_FEAR);`
and `events.ScheduleEvent(EVENT_FEAR, urand(8000, 12000), 0, PHASE_2);`